Gruntfile.js

```diff
@@ -2,10 +2,34 @@ module.exports = function(grunt) {
 
   // Project configuration.
   grunt.initConfig({
-      // TODO: add grunt-contrib-jst
-      // grunt.loadNpmTasks('grunt-contrib-uglify');
+    concat: {
+      dist: {
+        options: {
+          banner: "'use strict';\n",
+          process: function(src, filepath) {
+            return '// Source: ' + filepath + '\n' +
+              src.replace(/(^|\n)[ \t]*('use strict'|"use strict");?\s*/g, '$1');
+          },
+        },
+        files: {
+          'ooiui/static/js/compiled/science.js' : [
```

Comment:
so when we build out layouts and pages we can just specify the dependent JS files here. This replaces us having to explicitly import them and/or use require.js

Once we minify them, it'll be way faster too.